tools/update-icons

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
```

Comment:
If you switch to bash, a lot of the syntax can get prettier...

Reply:
Not very much. :-)

I have been writing sh scripts for autotools configure scripts and Makefile rules for the last good 20 years, so I know quite well how to write portable sh scripts, how to make them so readable even I can read them myself ten years later, and I only use bash in the rare case where array variables are very helpful.

Reply:
I go the other way, using bash everywhere except of I explicitly need posix compliant interoperability.

But I am fine with dropping bash entirely here.

Reply:
Added onto session-dbus-fixes and squashed into session-dbus-fixed, choose your preferred branch.

Reply:
In session-dbus.